examples/hello_world/finance/demo.py

```diff
@@ -0,0 +1,112 @@
+import time
```

Comment:
demo.py -> use your policy renaming the file

Reply:
simple buy sell

Reply:
ok